app/models/user.rb

```diff
@@ -20,14 +20,12 @@ def authenticate(params)
     end
 
     def create(params)
-      begin
-        api_response = RestClient::Request.execute(method: :get, url: "https://www.pivotaltracker.com/services/v5/me", user: params[:username], password: params[:password])
-        token = JSON.parse(api_response)["api_token"]
-      rescue Exception => err
-        puts err.response
-        return
-      end
-
+      api_response = RestClient::Request.execute(
+          method: :get,
+          url: "https://www.pivotaltracker.com/services/v5/me",
+          user: params[:username],
+          password: params[:password]) # HTTP errors are handled by application_controller.rb
+      token = JSON.parse(api_response)["api_token"]
```

Comment:
Better to be defensive here and use begin/rescue - but your formatting is much better here

Reply:
Oh sorry - didn't see your comment, good point!

Reply:
Actually, jsut tested it out, for some reason application_controller.rb doesn't handle if the user/pass is incorrect. So I'll keep the code as-is. The log is much better to look at anyways